src/components/FeatureViews/AdminPortal/AdminDashboard.js

```diff
@@ -171,6 +173,47 @@ const AdminDashboard = (props) => {
     { label: "Access Control" },
   ];
   
+  const QUERIES = {
```

Comment:
I forgot to note this last time - Don't copy what Neil did in Accounts for this. Since you're just making 1 query (GET_UNPAID_SESSIONS), just make a `const GET_UNPAID_SESSIONS` query and set that equal to `gql'query...`.